cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -58,7 +37,57 @@
         {%- set releases_url = routablepageurl(page.get_parent().specific, "latest_release") -%}
     {% endif %}
 
-    {%- set next_release_date_text = page.next_release_date|date("DATE_FORMAT") or _("To be announced") -%}
+    {# fmt:off #}
+    {% if page.release_date %}
+        {% set release_item = {
+                "term": release_date_label,
+                "descriptions": [
+                    {
+                        "description": page.release_date|date("DATE_FORMAT")
+                    }
+                ]
+            }
+        %}
+    {% else %}
+        {% set release_item = None %}
+    {% endif %}
+    {# fmt:on #}
+
+    {# fmt:off #}
+    {% if page.next_release_date %}
+        {% set next_release_item = {
+                "term": next_release_label,
+                "descriptions": [
+                    {
+                        "description": page.next_release_date|date("DATE_FORMAT") or _("To be announced")
```

Comment:
"To be announced" won't be picked up by Djanogs makemessages (ie. extract strings for translation)

Reply:
Ah good spot thank you 👍  updated